doc/releases/release-notes-2.0.rst

```diff
@@ -9,6 +9,7 @@ We are pleased to announce the release of Zephyr kernel version 2.0.0.
 
 Major enhancements with this release include:
 
+* The kernel is now 64 bit clean
```

Comment:
Is the meaning of this 100% clear? Just wondering - would we need to write a few more words here?

Reply:
Any proposal? :)

Reply:
Not really, I am not into the context of this at all, so I wouldn't risk any :)

Reply:
In marketing terms this would probably be "64 bit capable" :)

Reply:
At least, now, I would understand what you mean :)

Reply:
better would be to describe what this means, maybe "The kernel now supports both 32- and 64-bit architectures."